src/main/java/org/cryptomator/common/PluginClassLoader.java

```diff
@@ -19,6 +19,7 @@
 import java.util.EnumSet;
 import java.util.List;
 
+@Deprecated // to be moved to integrations-api 1.1.0
```

Comment:
Remark: The definition of the source to load plugins from should still reside here (the main app) and the integrations api should at most have a method accepting a Path/URL to search for plugins.